frontend/src/properties/PropertiesWebSocketContainerMachine.ts

```diff
@@ -217,6 +217,7 @@ export const propertiesWebSocketContainerMachine = Machine<
           selection.kind === 'Unknown' ||
           selection.kind === 'Diagram' ||
           selection.kind === 'Form' ||
+          selection.kind === 'ForceDirectedTree' || // Should be removed
```

Comment:
Can't you use RepresentationComponentRegistry#isRepresentation here?

Reply:
I don't think I can. The registry is not exported and I don't think I have access to `useContext` here.

Reply:
You can use `useContext` in `PropertiesWebSocketContainer` and indicate if the selection is a representation in the SwitchSelectionEvent.

Reply:
Done